src/openzaak/config/templates/admin/config_detail.html

```diff
@@ -75,6 +75,45 @@ <h1>{% trans 'Open Zaak API dashboard' %}</h1>
             </div>
         </section>
 
+        <!-- External API config -->
+        <section class="module">
+            <table class="table">
+                <caption>External services</caption>
+                <thead>
+                    <tr>
+                        <th scope="col">{% trans "Service" %}</th>
+                        <th scope="col">{% trans "Type" %}</th>
+                        <th scope="col">{% trans "Url" %}</th>
+                        <th scope="col">{% trans "NLX" %}</th>
+                        <th scope="col">{% trans "OAS" %}</th>
+                        <th scope="col">{% trans "Authorization" %}</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for service in external_services %}
+                    <tr>
+                        <td>{{ service.label }}</td>
+                        <td>{{ service.get_api_type_display }}</td>
```

Comment:
The type can be just the description. I want to get rid of the "ZRC", "ORC", etc. So, we can just show "Zaken" and "Overige" here instead of "ZRC (Zaken)" and "ORC (Overige)".

Reply:
`zgw_consumers.Service.api_type` field has choices like {"zrc": "ZRC (Zaken)", "orc":"ORC (Overige)" ...}
I can remove these abbreviations from choices labels in the `zgw_consumers` repo. Should I do it?

Reply:
oh weird, i saw in your screenshots this:

![image](https://user-images.githubusercontent.com/96970/79002978-ea330100-7b51-11ea-8f1e-ae2dfd9ac9d9.png)

Reply:
Oh, these ones are hardcoded in JS, I added template filter to make them consistent with detail page.